src/renderer/component/fileViewer/internal/player.jsx

```diff
@@ -259,8 +277,9 @@ class MediaPlayer extends React.PureComponent {
 
     // Files
     const isLoadingFile = !fileSource && isFileType;
+    const isLbryPackage = /application\/x(-ext)?-lbry$/.test(contentType);
```

Comment:
Can this just be replaced with the `this.supportedType()` function?

Reply:
See next comment